src/component/contents/DraftEditorBlock.react.js

```diff
@@ -121,8 +121,9 @@ class DraftEditorBlock extends React.Component<Props> {
         blockNode instanceof HTMLElement,
         'blockNode is not an HTMLElement',
       );
-      const blockBottom = blockNode.offsetHeight + blockNode.offsetTop;
-      const scrollBottom = scrollParent.offsetHeight + scrollPosition.y;
+      var blockBottom = blockNode.offsetHeight + blockNode.offsetTop;
+      var pOffset = scrollParent.offsetTop + scrollParent.offsetHeight;
+      var scrollBottom =  pOffset + scrollPosition.y;
```

Comment:
Hi @andpor, can you change these variable declarations to use `const` instead? Thanks!

Reply:
@claudiopro I just tried. Can you check how it looks?